mcp/src/main/java/io/modelcontextprotocol/server/McpServer.java

```diff
@@ -315,9 +311,7 @@ public AsyncSpecification tools(List<McpServerFeatures.AsyncToolSpecification> t
 		 */
 		public AsyncSpecification tools(McpServerFeatures.AsyncToolSpecification... toolSpecifications) {
 			Assert.notNull(toolSpecifications, "Tool handlers list must not be null");
-			for (McpServerFeatures.AsyncToolSpecification tool : toolSpecifications) {
-				this.tools.add(tool);
-			}
+			Collections.addAll(this.tools, toolSpecifications);
```

Comment:
Perhaps `this.tools.addAll(List.of(toolSpecifications))` would do the job without needing additional/external libraries?